### ISSUE TITLE
Cannot install supersafe with scala 2.12.2

### ISSUE THREAD

**Issue by timaschew** [scalatest/scalatest#1156]
I cannot install the supersafe library. As described on the website I added these lines to my project/plugins.sbt

```
addSbtPlugin("com.artima.supersafe" % "sbtplugin" % "1.1.2")
```

and this line to my ~/.sbt/0.13/global.sbt

`resolvers += "Artima Maven Repository" at "http://repo.artima.com/releases"`

I get this popup in IntelliJ

```
[warn] [FAILED ] com.artima.supersafe#supersafe_2.12.2;1.1.2!supersafe_2.12.2.jar(src): (0ms) 
[warn] ==== local: tried 
[warn] /Users/wilhelm/.ivy2/local/com.artima.supersafe/supersafe_2.12.2/1.1.2/srcs/supersafe_2.12.2-sources.jar 
[warn] ==== public: tried 
[warn] https://repo1.maven.org/maven2/com/artima/supersafe/supersafe_2.12.2/1.1.2/supersafe_2.12.2-1.1.2-sources.jar 
[warn] ==== Artima Maven Repository: tried 
[warn] http://repo.artima.com/releases/com/artima/supersafe/supersafe_2.12.2/1.1.2/supersafe_2.12.2-1.1.2-sources.jar 
[warn] :::::::::::::::::::::::::::::::::::::::::::::: 
[warn] :: FAILED DOWNLOADS :: 
[warn] :: ^ see resolution messages for details ^ :: 
[warn] :::::::::::::::::::::::::::::::::::::::::::::: 
[warn] :: com.artima.supersafe#supersafe_2.12.2;1.1.2!supersafe_2.12.2.jar(src) 
[warn] ::::::::::::::::::::::::::::::::::::::::::::::
```

This is my setup:
![scala-test-setup](https://cloud.githubusercontent.com/assets/110870/25820479/a1592488-3431-11e7-8d13-fbf4f675eb47.png)

Everything else is working fine.

maybe related to #1089

**Comment by timaschew** [comment]
It looks like there is no `source` version on http://repo.artima.com/releases/com/artima/supersafe/supersafe_2.12.2/1.1.2/

If I create a project in IntelliJ and uncheck the `Sources` checkbox at Scala Version it seems to work fine. BTW how can I verify if supersafe is really working?

**Comment by erikdstock** [comment]
Followup question here (sorry i do not know where else to put it, if it justifies its own issue, cannot find supersafe repo, etc)- all docs reference sbt 0.13, but would the directions be the same for sbt 1.0?

**Comment by cheeseng** [comment]
@timaschew SuperSafe is not open source, though its community edition which supports only scalactic and scalatest is free to use.

**Comment by cheeseng** [comment]
@erikdstock Yes, the steps should be similar, except instead of `~/.sbt/0.13/global.sbt` you will probably need to add the resolver in 1.0 one, fyi the sbt-plugin artifacts are available here: 

http://repo.artima.com/releases/com/artima/supersafe/sbtplugin_2.12_1.0/
